nerfstudio/viewer/app/src/modules/SidePanel/CameraPanel/CameraPropPanel.jsx

```diff
@@ -43,6 +54,20 @@ export default function CameraPropPanel(props) {
 
   const [, setControls] = useControls(
     () => ({
+      path: {
+        label: 'Export Name',
+        value: export_path,
+        onChange: (v) => {
+          const valid_filename_reg = /^([a-z]|[A-Z]|[0-9]|-|_|\(|\))+$/g;
+          if(!valid_filename_reg.test(v)){
+            alert("Please only use letters, numbers, parentheses, hyphens, and underscores in path name (no spaces either)");
```

Comment:
nit: "Please only use letters, numbers, parentheses, hyphens, and underscores in path name (no spaces either)" -> "Please only use letters, numbers, and hyphens)"